content/docs/pipelines/reference/api/kubeflow-pipeline-api-spec.html

```diff
@@ -173,12 +173,12 @@
 <div class="container">
     <h1>Kubeflow Pipelines API</h1>
     <p class="sw-info">
-        
-        Version: <span class="sw-info-version">0.1.23</span>
+
+        Version: <span class="sw-info-version">0.1.20</span>
```

Comment:
why to a lower version?

Reply:
The [swagger definition](https://github.com/kubeflow/pipelines/blob/master/backend/api/swagger/kfp_api_single_file.swagger.json) has the lower version number, which I thought odd as well. Do we need to correct it there?

Reply:
@OfficePop I updated the backend api version in swagger json. kubeflow/pipelines#2770 If you re-generate the api spec html, it should be using 0.1.38 instead of 0.1.20 now.

Reply:
Re-generated the api spec to account for this and other updates requested.

Reply:
Sorry for the delay @OfficePop! I didn't realize this PR is ready for another review!